docs/concept/TAP7.1-solution-design.md

```diff
@@ -0,0 +1,425 @@
+        EDCDB["EDC DB"]
+    end
+
+    subgraph "Capability Bundle: Digital Twin"
+        DTR["DTR"]
+        DTRDB["DTR DB"]
+    end
+
+    subgraph "Capability Bundle: Data Persitence Layer"
+        SimpleSubmodelServer["Simple Submodel Server"]
+    end
+
+    subgraph "Hausanschluss Bundles"
+        subgraph "Data Provider"
+            DataProvider["Data Provider"]
+        end
+        subgraph "Data Consumer"
+            DataConsumer["Data Consumer"]
+        end
+        subgraph "Enablement Provider"
+            EnablementProvider["Enablement Provider"]
+        end
+    end
+
+    EDC -.-> SSIWalletStub
+    EDC -.-> Vault
+    DataProvider --> EDC
+    DataProvider --> DTR
+    DataProvider --> SimpleSubmodelServer
+    DataConsumer --> EDC
+    EnablementProvider --> DataConsumer
+    EnablementProvider --> DataProvider
+
+````
+
+#### Deployment strategies
+
+##### Role: Data Provider
+
+The standard data provider role consists of a EDC, DTR and Submodel Server Bundles. EDC uses the Identity Wallet Bundle.
+
+````mermaid
+flowchart BT
+    A["Capability Bundle: Identity & Trust"]
+    B["Capability Bundle: Data Persistence Layer"]
+    C["Capability Bundle: Dataspace Connector"]
+    D["Capability Bundle: Digital Twin"]
+    E["Application to be tested/developed"]
+    E --> B
+    E --> C
+    E --> D
+    C --> A
+````
+
+##### Role: Data Consumer
+
+The standard data consumer role consists of only the EDC Bundle and the Identity Wallet Bundle.
+
+````mermaid
+flowchart BT
+    A["Capability Bundle: Identity & Trust"]
+    B["Capability Bundle: Dataspace Connector"]
+    C["Application to be tested/developed"]
+    C --> B
+    B --> A
+
+````
+
+##### Role: Enablement Provider
+
+The Enablement Provider roles consists of both the Data Consumer and Data Provider bundles. This allows provisioning of
+data and the consumtion of the provisioned data by a application.
+
+````mermaid
+flowchart BT
+    A["Capability Bundle: Identity & Trust"]
+    B["Capability Bundle: Data Persitance Layer"]
+    C["Application to be tested/developed"]
+    D["Capability Bundle: Digital Twin"]
+    E["Capability Bundle: Dataspace Connector (Provider)"]
+    F["Capability Bundle: Dataspace Connector (Consumer)"]
+    C --> B
+    C --> D
+    C --> E
+    C --> F
+    E --> A
+    F --> A
+````
+
+##### Bring you own
+
+Each Bundle can be deactivated to use your own component. This requires configuration of the component specific
+parameters in the other components.
+
+In this example this would be the configuration of the Identity Wallet within the EDC.
+
+````mermaid
+flowchart BT
+    A["Application to be tested/developed"]
+    B["Capability Bundle: Dataspace Connector"]
+    C["<s>Capability Bundle: Identity & Trust</s>"]
+    D["Own Identity Wallet"]
+    A --> B
+    B -- identitywallet - bundle . enabled = false --> C
+    B -- configure --> D
+
+````
+
+## 4.4 System
+
+### 4.4.1 Context Delimitation / 4.4.2 Interface Documentation
+
```

Comment:
Any kind of prosa documentation to the chart? Explaining the chart and what we will change here?

Reply:
added